sign/error-codes.md

```diff
@@ -1,4 +1,4 @@
-# Codes
+# Error Codes
```

Comment:
We renamed Error Codes to Codes because we've added `case .userDisconnected: return 6000`, which is technically not an error.

Reply:
Remember we actually used to call these Reason Codes before beta.100... I can't remember why we stopped calling them like that

Reply:
Honestly let's go with this for now and then we can revisit